static/app/components/buttonClick.tsx

```diff
@@ -0,0 +1,55 @@
+import {useCallback} from 'react';
+
+import HookStore from 'sentry/stores/hookStore';
+
+const useGetHook = ({disabled, busy, onClick}) => {
+  const useButtonTracking = HookStore.get('react-hook:track-buttons')[0];
+  const useButtonClick = () =>
+    useCallback((e: React.MouseEvent) => {
```

Comment:
@roggenkemper So a lot of this logic is duplicated in getsentry, not sure if there is a way to encapulate the fallback logic into a hook that getsentry can use as well

Reply:
@roggenkemper nvm, i think the hook is slightly different so let's just leave it as-is